src/main/kotlin/blackjack/Main.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack
+
+import blackjack.model.BlackJack
+import blackjack.model.Player
+import blackjack.model.Players
+import blackjack.view.Input
+import blackjack.view.Output
+
+fun main() {
+    val names = Input.names()
+    val players = Players(names.map { Player(it) })
+    val blackJack = BlackJack(players)
+    Output.players(players)
+    blackJack.firstTurn()
+    for (count in 0 until players.size()) {
+        Output.pickResult(players.findPlayer(count))
+    }
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
    repeat(players.size()) {
        Output.pickResult(players.findPlayer(it))
    }
```